freeciv-proxy/civcom.py

```diff
@@ -142,6 +142,7 @@ def send_buffer_append(self, data):
                 data.decode(
                     encoding="utf-8",
                     errors="ignore"))
+            logger.info("send to client: " + self.send_buffer[-1])
         except UnicodeDecodeError:
```

Comment:
Can you please  change so that this logging of messages are not logged by default?